prettypandas/styler.py

```diff
@@ -202,7 +225,7 @@ def min(self, title="Minimum", **kwargs):
         """
         return self.summary(np.min, title, **kwargs)
 
-    def as_percent(self, subset=None, precision=None, locale=None):
+    def as_percent(self, subset=None, precision=0, **kwargs):
```

Comment:
I'm still not sure about how `as_percent` should be handled. I'll get back to you on it